src/epp_topics/scientific_computing/optimagic_features/objectives_materials.ipynb

```diff
@@ -4,21 +4,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Estimagic overview\n",
+    "# Optimagic overview\n",
     "\n",
     "## Learning Objectives\n",
     "\n",
     "After working through this topic, you should be able to:\n",
     "\n",
-    "- Explain how estimagic makes numerical optimization more user friendly for non-experts\n",
-    "- List several diagnostic features of estimagic\n",
+    "- Explain how optimagic makes numerical optimization more user friendly for non-experts\n",
+    "- List several diagnostic features of optimagic\n",
     "- Explain why representing parameters as flat vectors is not a good idea in complex\n",
     "applications\n",
     "\n",
     "## Materials\n",
     "\n",
-    "The following is an estimagic presentation from the euro-scipy conference. You don't\n",
-    "have to understand all details. Just try to get an overview of the main estimagic \n",
+    "The following is an optimagic presentation from the euro-scipy conference. You don't\n",
+    "have to understand all details. Just try to get an overview of the main optimagic \n",
     "features.\n",
     "\n",
     "<iframe width=\"560\" height=\"315\" src=\"https://www.youtube.com/embed/5xYn0v1zEsY?si=ziL7u99JBXiT58yA\" title=\"YouTube video player\" frameborder=\"0\" allow=\"accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share\" allowfullscreen></iframe>\n"
```

Comment:
Wouldn't the 2024 one be better?

Reply:
Yes, but as far as I know, the EuroScipy 2024 videos have not been uploaded yet.